src/location-search/map-selection/index.tsx

```diff
@@ -64,14 +64,12 @@ const MapSelection: React.FC<Props> = ({
     ],
   );
 
-  const {locations, isSearching, error} = useReverseGeocoder(
+  const {closestLocation: location, isSearching, error} = useReverseGeocoder(
     centeredCoordinates,
   );
 
   const {location: geolocation} = useGeolocationState();
 
-  const location = locations?.[0];
```

Comment:
Weirdly enough here it's correct. Might be why it hasn't been discovered, seeing as "Map"-selection is more visual and precise.